openmdao/docs/feature_reference/core_features/working_with_derivatives/approximating_totals.rst

```diff
@@ -0,0 +1,52 @@
+.. _feature_declare_totals_approx:
+
+
+Approximating Semi-Total Derivatives
+====================================
+
+There are also times where it makes more sense to approximate the derivatives for an entire group in one shot. You can turn on
```

Comment:
suggest removing "also" and "more" so the doc page stands on it's own

Reply:
Fixed